frontend/src/metabase/actions/containers/ActionExecuteModal/ActionExecuteModal.tsx

```diff
@@ -41,13 +49,21 @@ const ActionExecuteModal = ({
     [action, onSubmit],
   );
 
+  const handleSubmitSuccess = useCallback(() => {
+    onClose();
```

Comment:
nit: with modern js you can have `onClose?.()`, which allows you do have `onClose` not defined

Reply:
6eca96e23975a56ab12d85f163198060f07b0934